Framework/PythonInterface/plugins/algorithms/ComputeCalibrationCoefVan.py

```diff
@@ -0,0 +1,168 @@
+from mantid.api import PythonAlgorithm, AlgorithmFactory, WorkspaceProperty, Progress, InstrumentValidator
+from mantid.kernel import Direction
+import mantid.simpleapi as api
+import numpy as np
+from scipy import integrate
+import scipy as sp
+import mlzutils
```

Comment:
No, it is not facility specific. Other algorithms in the mlzutils either. I've called it mlzutils because it is used now mainly for DNS and TOFTOF@MLZ, but I can rename it.  How to name it? May be utils?